extensions/sql/build.gradle.kts

```diff
@@ -0,0 +1,35 @@
+/*
+ *  Copyright (c) 2022 Daimler TSS GmbH
```

Comment:
Probably a copy/paste error on the headers

Reply:
yup - fixed